apps/docs/docs/examples/other_llms.mdx

```diff
@@ -0,0 +1,41 @@
+import CodeBlock from "@theme/CodeBlock";
+import CodeSource from "!raw-loader!../../../../examples/mistral";
+
+# Using other LLM APIs
```

Comment:
FYI one super common question I see on discord is how to use azure modrld

Not sure if that's in the examples folder or should have a dedicated spot in the docs or not

Reply:
Good feedback! I will include it in the next rev somewhere, this believe it or not is just a WIP.